### PR TITLE
Build improvements and dependency updates

### PR DESCRIPTION
**Pull request by maoo**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,8 +4,8 @@ lein: 2.8.1
 jdk:
   - oraclejdk8
   - oraclejdk9
-  - oraclejdk10
-  - oraclejdk11
+  # - openjdk10   # TravisCI deprecated their Oracle JDK 10 image, annoyingly...
```
Comment (pmonks) [inline .travis.yml]: This should not be commented out, since line 20 works around TravisCI's terrible base openjdk10 image.

File: project.clj [hunk 2]
```diff
@@ -46,7 +46,8 @@
                       [org.jolokia/jolokia-jvm                   "1.6.0" :classifier "agent"]
                       [clj-time                                  "0.15.1"]
                       [com.github.grinnbearit/freemarker-clj     "-SNAPSHOT"]
-                      [irresponsible/tentacles                   "0.6.2"]
+                      [irresponsible/tentacles                   "0.6.2" :exclusions [org.apache.ant/ant]]
+                      [org.apache.ant/ant                        "1.10.5"]
```
Comment (pmonks) [inline project.clj]: A better solution here is to force an upgrade of [`pathetic` to v0.5.1](https://clojars.org/pathetic/versions/0.5.1).  

That version doesn't have a dependency on ClojureScript, which thereby also completely removes the dependency on ant (a vastly better solution).

File: .travis.yml (hunk 1 above)
Comment (maoo) [reply .travis.yml]: my bad, I reverted the change

File: project.clj [hunk 3]
```diff
@@ -47,6 +47,7 @@
                       [clj-time                                  "0.15.1"]
                       [com.github.grinnbearit/freemarker-clj     "-SNAPSHOT"]
                       [irresponsible/tentacles                   "0.6.2"]
```
Comment (pmonks) [inline project.clj]: You'll need to add an exclusion for `pathetic` here, to avoid conflicting versions.

**Pull request merged**